docs/core/tools/dotnet-nuget-push.md

```diff
@@ -68,6 +68,10 @@ Prints out a short help for the command.
 
   Specifies the server URL. This option is required unless `DefaultPushSource` config value is set in the NuGet config file.
 
+* **`--skip-duplicate`**
+
+  When pushing multiple packages to an HTTP(S) server, treat any 409 Conflict response as a warning so that the push can continue.
```

Comment:
```suggestion
  When pushing multiple packages to an HTTP(S) server, treat any 409 Conflict response as a warning so that the push can continue. Available since .NET Core 3.1 SDK.
```